packages/eslint-config-fb-strict/package.json

```diff
@@ -10,7 +10,7 @@
     "eslint-config-fbjs": "^2.0.0"
   },
   "peerDependencies": {
-    "babel-eslint": "^7.2.3",
+    "babel-eslint": "^7.2.3 || ^8.0.0",
```

Comment:
doesn't really help us, as `eslint-config-fbjs` requires 7. But it's a step on the way, at least

Reply:
We could also make it `">=7.2.3"`, no?

Reply:
it's just a peer, doesn't really matter. And 8 is actually not breaking, it just updates to babel 7 under the hood. 9 might be breaking, though 🙂